packages/core/src/Utils/pwa/register-service-worker.js

```diff
@@ -1,4 +1,4 @@
-const EVERY_HOUR = 36000000; // 1000 * 60 * 60
+const EVERY_HOUR = 3600000; // 1000 * 60 * 60
```

Comment:
@reza-fs :D

Reply:
FYI: Now we can write big numbers in JS with `_` separator for better readability:

```
let num = 3_600_000; // this is same as 3600000 but easier to read.
```